src/ProjectTemplates/Web.Spa.ProjectTemplates/content/Angular-CSharp/ClientApp/package.json

```diff
@@ -32,17 +32,17 @@
     "zone.js": "0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.803.24",
-    "@angular/cli": "8.3.14",
-    "@angular/compiler-cli": "8.2.12",
-    "@angular/language-service": "8.2.12",
+    "@angular-devkit/build-angular": "^0.803.26",
+    "@angular/cli": "^8.3.26",
+    "@angular/compiler-cli": "^8.2.14",
+    "@angular/language-service": "^8.2.12",
     "@types/jasmine": "~3.4.4",
     "@types/jasminewd2": "~2.0.8",
     "@types/node": "~12.11.6",
     "codelyzer": "^5.2.0",
     "jasmine-core": "~3.5.0",
     "jasmine-spec-reporter": "~4.2.1",
-    "karma": "^4.4.1",
+    "karma": "^5.0.2",
```

Comment:
Bit surprised we can bump the major version without adjusting some of our code 😃